<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICT_UR10</class>
 <widget class="QMainWindow" name="ICT_UR10">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ICT_UR10</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEditResult">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="labelResultList">
      <property name="text">
       <string>Test Result List</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelSN">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Scan SN:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditSN">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelScanStatus">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Scanner Status:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelRobotStatus">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Robot Status:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="labelResult">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Result:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QTableWidget" name="tableWidgetResultList">
      <column>
       <property name="text">
        <string>No.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Scan SN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Scan Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Test Result</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParameter">
    <property name="title">
     <string>Parameter</string>
    </property>
    <addaction name="actionScanner"/>
    <addaction name="actionRobot"/>
    <addaction name="separator"/>
    <addaction name="actionCommunication"/>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menuParameter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScanner"/>
   <addaction name="actionRobot"/>
   <addaction name="actionCommunication"/>
   <addaction name="actionLogin"/>
  </widget>
  <action name="actionScanner">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/scanner_barcode_32px.png</normaloff>:/res/scanner_barcode_32px.png</iconset>
   </property>
   <property name="text">
    <string>Scanner</string>
   </property>
  </action>
  <action name="actionRobot">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Industrial_robot_32px.png</normaloff>:/res/Industrial_robot_32px.png</iconset>
   </property>
   <property name="text">
    <string>Robot</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Login_32px.png</normaloff>:/res/Login_32px.png</iconset>
   </property>
   <property name="text">
    <string>LogIn</string>
   </property>
   <property name="toolTip">
    <string>LogIn</string>
   </property>
  </action>
  <action name="actionCommunication">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/communication_chat_32px.png</normaloff>:/res/communication_chat_32px.png</iconset>
   </property>
   <property name="text">
    <string>Communicate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
